zen/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,11 @@
+"Current Challenge" = "Current Challenge";
```

Comment:
better to create a key with a better naming including flow, screen and element like 
https://github.com/TouchInstinct/team/blob/master/processes/Strings-And-Images-Naming-Rules.md

Reply:
done

Reply:
done